vhdbuilder/packer/windows-vhd-builder-gen2.json

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
It seems like that we can reuse vhdbuilder/packer/windows-vhd-builder-sig.json

Reply:
Not exactly. We have a `"gen2_captured_sig_version"` that will be used when converting from packer SIG to the VHD for publish pipeline. Also, since we don't want to reuse `sig_image_name`, the image name for the packer output in `shared_image_gallery_destination` should be something like `gen2_sig_image_name`.